jackrabbit-core/src/test/java/org/apache/jackrabbit/core/security/user/UserImporterTest.java

```diff
@@ -1671,6 +1671,16 @@ public Node getNodeOrNull(String absPath) throws RepositoryException {
             return null;
         }
 
+        @Override
+        public String getExpandedName(Item item) throws RepositoryException {
+            throw new RepositoryException("see JCR-5137");
```

Comment:
why this? If not necessary rather throw `OperationUnsupportedException`

Reply:
ack, I can do that